### ISSUE TITLE
Loop cut and Bevel

### ISSUE THREAD

**Issue by omariparris** [mlfarrell/VertoJS-VertoStudio3D#1]
A VertoJS code for loop cut and bevel would be nice

**Comment by mlfarrell** [comment]
I agree. If anyone wants to implement either of these and PR them into this repo, I’d be happy to include it here.
